bank/views.py

```diff
@@ -52,8 +52,8 @@ def get(self, request):
 
     @method_decorator(login_required)
     def post(self, request):
-        print 'IN POST'
-        print request.POST
+        print('IN POST')
+        print(request.POST)
```

Comment:
Blimey! There's a lot of debugging.

This could be an opportunity to remove some of the more useless debugging statements, like this one.

I like how you've converted most of them to `logger.debug(...)` so they can at least be suppressed in production.